docs/stock/understanding-highcharts-stock.md

```diff
@@ -5,30 +5,92 @@ Highcharts Stock is based on Highcharts, meaning it has all the core functionali
+
+See [Navigator](https://highcharts.com/docs/stock/navigator) for more information.
+
+<iframe style="width: 100%; height: 640px; border: none;" src=https://highcharts.github.io/highcharts-utils/samples/#gh/352ea24215ec4f79bb50ead0152430087bc3551e/sample/stock/interactive-docs/navigator allow="fullscreen"></iframe>
+
+```js
+chart.update({
+    navigator: {
+        enabled: true,
+        height: 100
+    },
+    scrollbar: {
+        enabled: false
+    }
+});
+```
+Navigator and scrollbar does not have its own update method, use [Chart update](https://api.highcharts.com/class-reference/Highcharts.Chart#update) instead. See [navigator API options](https://api.highcharts.com/highstock/navigator).
 
-See [Navigator](https://highcharts.com/docs/stock/navigator) for more information.
 
 Range selector
 --------------
 
 Allows you to quickly select a range to be shown on the chart or specify the exact interval to be shown.
 
+<iframe style="width: 100%; height: 600px; border: none;" src=https://highcharts.github.io/highcharts-utils/samples/#gh/352ea24215ec4f79bb50ead0152430087bc3551e/sample/stock/interactive-docs/range-selector allow="fullscreen"></iframe>
+
 See [Range selector](https://highcharts.com/docs/stock/range-selector) for more information.
 
-Scrollbar
+```js
+chart.update({
+    rangeSelector: {
+        enabled: true,
+    }
+});
+```
+
+Range selector does not have its own update method, use [Chart update](https://api.highcharts.com/class-reference/Highcharts.Chart#update) instead. See [range selector API options](https://api.highcharts.com/highstock/rangeSelector).
+
+Crosshair
 ---------
 
-Allows scrolling on the chart.
+Shows a line following the tooltip of a chart to better read results of the axes. This functionality can be found in the [axis API](https://api.highcharts.com/highstock/xAxis.crosshair) options. Crosshairs can also be used in Highcharts, but are not enabled by default.
 
-See [Scrollbar](https://highcharts.com/docs/chart-concepts/scrollbar) for more information.
+<iframe style="width: 100%; height: 600px; border: none;" src=https://highcharts.github.io/highcharts-utils/samples/#gh/352ea24215ec4f79bb50ead0152430087bc3551e/sample/stock/interactive-docs/crosshair allow="fullscreen"></iframe>
 
-Crosshair
+```js
+chart.xAxis[0].update({
+    crosshair: {
+        snap: false
+    }
+});
+```
+
+Crosshair is an axis property, therefore update it via [Axis update](https://api.highcharts.com/class-reference/Highcharts.Axis#update). See [axis API options](https://api.highcharts.com/highstock/xAxis.crosshair).
+
+Data grouping
 ---------
 
-Shows a line following the tooltip of a chart to better read results of the x-axis. This functionality can be found in the [Tooltip](https://highcharts.com/docs/chart-concepts/tooltip) option. Crosshairs can also be used in Highcharts, but are not enabled by default.
+Changes granularity of the data depending on the zoom level, data density and user options.
```

Comment:
I don't really understand this sentance. Could you try making it more clear? Thanks!